spec/reek/smells/too_many_instance_variables_spec.rb

```diff
@@ -58,7 +58,20 @@ def bravo
       end
     EOS
 
-    expect(src).not_to reek_of(:TooManyInstanceVariables).with_config(config)
+    expect(src).not_to reek_of(:TooManyInstanceVariables)
+  end
+
+  it 'reeks of unneeded supression' do
+    src = <<-EOS
```

Comment:
Are you sure it succeeds without the changes?

I think the test would be much improved if it added an expectation on the smell message.

To make this more a test about SmellDetector, it should be moved to the `smell_detector_spec.rb` file.

The `sniff` method is not what is tested here.